toolbox/githubctl/main.go

```diff
@@ -212,10 +212,13 @@ func UpdateIstioVersionAfterReleaseTagsMadeOnDeps() error {
 }
 
 // CreateIstioReleaseUploadArtifacts creates a release on istio and uploads dependent artifacts
-func CreateIstioReleaseUploadArtifacts() error {
+func CreateIstioReleaseUploadArtifacts(refSHA string) error {
 	assertNotEmpty("base_branch", baseBranch)
 	assertNotEmpty("next_release", nextRelease)
 	releaseTag, err := getReleaseTag()
+	if releaseTag == *nextRelease {
+		return fmt.Errorf("Next Release tag needs to be greater than the current release.")
```

Comment:
error should start with lower case letter

Reply:
Done